images/dind/build.sh

```diff
@@ -1,3 +1,17 @@
 set -ex
 
 docker build --build-arg HTTPS_PROXY --build-arg HTTP_PROXY -t fnproject/dind:latest .
+
+# Match version with Docker version
+version=$(docker run --rm -v "$PWD":/app treeder/bump  --extract --input "`docker -v`")
```

Comment:
this uses the docker version from `treeder/bump` ? is this right?

Reply:
Ye-es. Ish. What we don't do (at the moment) is bump the dind:17.12 requirement in the fnserver dockerfile.

I *think* the thing to do here is to tag and push an fnproject/dind:latest - *and* to refer to that from the fnserver dockerfile.

Reply:
(pushed a second commit with that change too)